packages/protocol/test-sol/stability/SortedOracles.t.sol

```diff
@@ -0,0 +1,592 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.5.13;
+pragma experimental ABIEncoderV2;
+
+import { Test } from "celo-foundry/Test.sol";
+import { SortedOracles } from "../../contracts/stability/SortedOracles.sol";
+import "../../contracts/common/FixidityLib.sol";
+import "../../contracts/common/linkedlists/AddressSortedLinkedListWithMedian.sol";
+import "../../contracts/common/linkedlists/SortedLinkedListWithMedian.sol";
+import { Constants } from "../constants.sol";
+
+contract SortedOraclesTest is Test, Constants {
```

Comment:
Not sure if intentional, but there isnt a test for the `setBreakerBox()` function

Reply:
It's already on Mento's

Reply:
I saw it in the mento test, but according to your comment, it's not realy clear yet if the mento test will stay long term. Might be best to add the breaker tests to this file as well.